javascript/organizing_your_javascript_code/webpack.md

```diff
@@ -42,6 +42,16 @@ const HtmlWebpackPlugin = require('html-webpack-plugin');
 // ...
 ```
 
+<div class="lesson-note lesson-note--tip" markdown="1">
+
+#### A note if your fonts are not loading
+
+  If you have any issues loading fonts with webpack, the issue is not with webpack, but how you declare the css rule.
```

Comment:
Sorry, @LexiphanicStroon, I should have caught this earlier. We can ignore the other linting errors, but should be good citizens and make sure the new content is passing lint checks. 

Thankfully, it only needs a some capitalisation:

```suggestion
  If you have any issues loading fonts with Webpack, the issue is not with Webpack, but how you declare the CSS rule.
```

Reply:
No need to apologize Kevin I greatly appreciate you pointing it out!